src/sharing/share.styl

```diff
@@ -11,6 +11,9 @@
         cursor pointer
 
 :local
+    .share-modal
+        max-width 33rem !important // waiting for modal size presets in cozy-ui
```

Comment:
That is not really clean.
I discussed with people about this and I told them that it is not third-app risponsibility.

There is a reason we build an UI library.
If this is important, then we should release a brand new version of the library with "size presets".

That said, the commit message is really fun 🍆